### PR TITLE
Jetpack plans: new descriptions test

### PR DESCRIPTION
**Pull request by johnHackworth**
As proposed in`p7kMJG-1pI-p2`, we are going to run a test that changes the descriptions of the features on the jetpack plans pages.

This is how the test looks like:
image

image

How to test
====

1. Go to http://calypso.localhost:3000/jetpack/connect/store and check the old plans page style and make sure is still three
2. Open the developer console and run `localStorage.setItem( 'ABTests', '{"jetpackNewDescriptions_20170327":"showNew"}' );`
3. Reload the page
4. The new descriptions & look should be there
5. Go to http://calypso.localhost:3000/ and select a jetpack site. Go to the plans page. You should see the new Style & descriptions there too

### CODE REVIEW

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -282,19 +326,27 @@ export const PLANS_LIST = {
 		getStoreSlug: () => PLAN_JETPACK_PERSONAL,
 		availableFor: ( plan ) => includes( [ PLAN_JETPACK_FREE ], plan ),
 		getPathSlug: () => 'jetpack-personal',
-		getDescription: () => i18n.translate(
-			'Essentials for every site. The most affordable solution to keep your' +
-			' personal or small business site backed up and spam-free.'
+		getDescription: ( abtest ) => abtest && abtest( 'jetpackNewDescriptions' ) === 'showNew'
+			? i18n.translate( 'Security essentials for every site.' )
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Security essentials for every site' )` *ES Score: 21*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -212,11 +222,19 @@ export const PLANS_LIST = {
 		getTitle: () => i18n.translate( 'Free' ),
 		getProductId: () => 2002,
 		getStoreSlug: () => PLAN_JETPACK_FREE,
-		getDescription: () => i18n.translate(
-			'The features most needed by WordPress sites' +
-			' — perfectly packaged and optimized for everyone.'
+
+		getDescription: ( abtest ) => abtest && abtest( 'jetpackNewDescriptions' ) === 'showNew'
+			? i18n.translate( 'Everything you need to get started.' )
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Everything you need to get started' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by richardmuscat** [comment]
@johnHackworth the setup looks great. The only things that are needed are applying the correct text/features in the correct order based on #12054 mockup and some style changes:

- String changes: listed out below.
- Remove check marks
- Make the feature description (the part that's not bold) a slightly smaller font size

**Free Plan**

- Sub heading: GOOD

1. Core Jetpack Services / Stats, themes, and promotion tools.
2. Basic Security / Brute force protection, monitoring, secure logins, updates.
3. Basic Support / Free support to help you make the most of Jetpack.
4. Speed and Storage / Unlimited use of our high speed image content delivery network.

**Personal Plan**

- Sub heading: GOOD

1. Core Jetpack Services / Stats, themes, and promotion tools.
2. Essential Security / Daily backups, unlimited storage, one-click restores, spam filtering.
3. Priority Support / Faster response times from our security experts.
4. Speed and Storage / Unlimited use of our high speed image content delivery network.

**Premium Plan**

- Sub heading: Earn income and reduce costs.

1. Core Jetpack Services / Stats, themes, and promotion tools.
2. Advanced Security / Also includes daily scans for malware and security threats.
3. Priority Support / Faster response times from our security experts.
4. Speed and Storage / Also includes 13Gb of high-speed, ad-free video hosting.
5. Revenue Generation / High quality ads to generate income from your site.
6. Advanced Traffic Tools / Automatically re-promote existing content to social media.

**Professional Plan**

- Sub heading: GOOD

1. Core Jetpack Services / Stats, themes, and promotion tools.
2. Advanced Security / Also includes real-time backup and automated threat resolution.
3. Priority Support / Faster response times from our security experts.
4. Speed and Storage / Also includes unlimited, high-speed, ad-free video hosting.
5. Revenue Generation / High quality ads to generate income from your site.
6. Advanced Traffic Tools / Also includes SEO previews and Google Analytics.

File: client/lib/plans/constants.js [hunk 3]
```diff
@@ -752,6 +842,130 @@ export const FEATURES_LIST = {
...
+		getSlug: () => FEATURE_PRIORITY_SUPPORT_JETPACK,
+		getTitle: () => i18n.translate( 'Priority Support' ),
+		getDescription: () => i18n.translate(
+			'Faster response times from our security experts.'
+		),
+		hideInfoPopover: true
+	},
+	[ FEATURE_TRAFFIC_TOOLS_JETPACK ]: {
+		getSlug: () => FEATURE_TRAFFIC_TOOLS_JETPACK,
+		getTitle: () => i18n.translate( 'Advanced Traffic Tools' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Popular Traffic Tools' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 4]
```diff
@@ -752,6 +842,130 @@ export const FEATURES_LIST = {
...
+			'13Gb of high-speed, HD, and ad-free video hosting.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_SECURITY_ESSENTIALS_JETPACK ]: {
+		getSlug: () => FEATURE_SECURITY_ESSENTIALS_JETPACK,
+		getTitle: () => i18n.translate( 'Essential Security' ),
+		getDescription: () => i18n.translate(
+			'Daily backups, unlimited storage, one-click restores, spam filtering.'
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Daily backups, unlimited storage, one-click restores, and spam filtering.' )` *ES Score: 20*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 5]
```diff
@@ -752,6 +842,130 @@ export const FEATURES_LIST = {
...
+		getTitle: () => i18n.translate( 'Video Hosting' ),
+		getDescription: () => i18n.translate(
+			'13Gb of high-speed, HD, and ad-free video hosting.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_SECURITY_ESSENTIALS_JETPACK ]: {
+		getSlug: () => FEATURE_SECURITY_ESSENTIALS_JETPACK,
+		getTitle: () => i18n.translate( 'Essential Security' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Security Essentials' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-plans-descriptions-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 6]
```diff
@@ -752,6 +842,130 @@ export const FEATURES_LIST = {
...
+			'Also includes daily scans for malware and security threats.'
+		),
+		hideInfoPopover: true
+	},
+
+	[ FEATURE_REVENUE_GENERATION_JETPACK ]: {
+		getSlug: () => FEATURE_REVENUE_GENERATION_JETPACK,
+		getTitle: () => i18n.translate( 'Revenue Generation' ),
+		getDescription: () => i18n.translate(
+			'High quality ads to generate income from your site.'
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Display high-quality ads to generate income directly from your site.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 7]
```diff
@@ -752,6 +842,130 @@ export const FEATURES_LIST = {
 			'Custom polls, surveys, ratings, and quizzes for the ultimate in customer and reader engagement.'
 		)
 	},
+
+	[ FEATURE_CORE_JETPACK ]: {
+		getSlug: () => FEATURE_CORE_JETPACK,
+		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getDescription: () => i18n.translate(
+			'Stats, themes, and promotion tools.'
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Stats, customization, and promotion tools' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js (hunk 4 above)
Comment (oskosk) [inline client/lib/plans/constants.js]: Isn't this suggestion appropriate ?

**Comment by mattwondra** [comment]
@johnHackworth It looks like the `public/directly.css` file was added to this PR — I just wanted to double-check, was this intentional? That file is compiled on build and should be left out of the repo.

**Pull request merged**